docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -438,3 +438,22 @@ An external {jdgserver_name} deployment is supported for multi-site setups as ou
 
 The Oracle Database JDBC driver is no longer part of the Keycloak distribution.
 If you wish to use Oracle DB, you must manually install a version of the Oracle Driver that is compatible with your specific environment. Instructions for this process can be found in the https://www.keycloak.org/server/db[Configuring the database] {section}.
+
+= Deprecated theme variables
+
+The following variables were deprecated in the Account theme:
+
+* `authUrl`. Use `authServerUrl` instead.
+
+The following variables from the environment script injected into the page of the Account theme are deprecated:
+
+* `authUrl`. Use `authServerUrl` instead.
+* `features.isInternationalizationEnabled`. Do not use this variable.
+
+The following variables were deprecated in the Admin theme:
+
+* `authUrl`. Do not use this variable.
```

Comment:
are you sure about this one? Is it true @vmuzikar, that we only need one url in the admin ui?

Reply:
It is not referenced anywhere in the templates or JS code.

Reply:
Yes it was used https://github.com/keycloak/keycloak/pull/30125/files#diff-59cd7df86bacac814b984d485e647e3db6a6011e45a75fbc863bef9d748842fdL27 both `authUrl` and `authServerUrl` are used in the admin ui